pkg/gomod/release_meta.go

```diff
@@ -26,7 +26,7 @@ import (
 	"knative.dev/test-infra/pkg/golang"
 )
 
-// ReleaseMeta holds metadata important to module release status.
+// ReleaseMeta holds meta data important to module release status.
```

Comment:
this got reverted somehow?

Reply:
matt wanted it to be two words

Reply:
oooh oops, matt wanted it to be one word... one sec 🤕

Reply:
lol

Reply:
😁